### ISSUE TITLE
Dropdown not working @4.6.0/dist/united

### ISSUE THREAD

**Issue by dianekaplan** [thomaspark/bootswatch#1117]
I'm using bootstrap CDN and I'm so excited to have found this theme!

Unfortunately when I copy/paste the sample code for the gray navbar [here](https://bootswatch.com/united/), nothing happens when I click the down arrow for the dropdown field. 
`<nav class="navbar navbar-expand-lg navbar-light bg-light">
  <div class="container-fluid">
    <a class="navbar-brand" href="#">Navbar</a>
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarColor03" aria-controls="navbarColor03" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarColor03">
      <ul class="navbar-nav me-auto">
        <li class="nav-item">
          <a class="nav-link active" href="#">Home
            <span class="visually-hidden">(current)</span>
          </a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="#">Features</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="#">Pricing</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="#">About</a>
        </li>
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Dropdown</a>
          <div class="dropdown-menu">
            <a class="dropdown-item" href="#">Action</a>
            <a class="dropdown-item" href="#">Another action</a>
            <a class="dropdown-item" href="#">Something else here</a>
            <div class="dropdown-divider"></div>
            <a class="dropdown-item" href="#">Separated link</a>
          </div>
        </li>
      </ul>
      <form class="d-flex">
        <input class="form-control me-sm-2" type="text" placeholder="Search">
        <button class="btn btn-secondary my-2 my-sm-0" type="submit">Search</button>
      </form>
    </div>
  </div>
</nav>`

Here are my css and js files:

```
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootswatch@4.6.0/dist/united/bootstrap.css">
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootswatch@4.6.0/dist/united/bootstrap.min.css">

        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
```

Is there a different javascript file (or something else) I need to make the toggle work correctly in the example code?  I tried substituting the files listed in [this similar ticket](https://github.com/thomaspark/bootswatch/issues/1086), but no luck. 

Thanks in advance!
Diane

**Comment by dianekaplan** [comment]
got it!  The sample navbar code does work if I swap in the 5.0.2 version of the bootstrap js: 
https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js

**Comment by thomaspark** [comment]
Hey Diane, glad you got it working. If you run into any other issues with the theme, please let us know!

**Comment by ayushnoori** [comment]
Ran into the same issue today, it appears that Bootswatch is incompatible with the [template code](https://getbootstrap.com/docs/4.1/getting-started/introduction/#starter-template) available on the Bootstrap website? Thanks @dianekaplan for the fix!

**Comment by ekrichbaum** [comment]
a little related and a little not.  That was bootstrap 4.0 not 4.6.  You may want to try just matching up to see if your issue goes away, or use Bootstrap 5 with the matching Bootswatch 5.  Note that there are some differences in style names to deal with but matching up will always be a good place to start.

**Comment by ayushnoori** [comment]
Sounds great, will investigate. Thanks, @ekrichbaum.
